x-pack/test/security_solution_cypress/cypress/tasks/timeline.ts

```diff
@@ -306,7 +306,14 @@ export const closeTimeline = () => {
 };
 
 export const createNewTimeline = () => {
-  cy.get(NEW_TIMELINE_ACTION).click();
+  recurse(
+    () => {
+      cy.get(NEW_TIMELINE_ACTION).click();
+      return cy.get(CREATE_NEW_TIMELINE).first();
+    },
+    (sub) => sub.is(':visible')
+  );
+
```

Comment:
is this change still necessary? In theory Cypress is supposed to check for an element to be visible before clicking on it